### ISSUE TITLE
How does an unbalanced data set impact on the performance of Watershed

### ISSUE THREAD

**Issue by HarryShen668** [BennyStrobes/Watershed#3]
Hi,Thanks for this great work!
I‘d like to ask **Does an unbalanced data set have a big impact on the performance of this model**? Because the outliers themselves are relatively rare in the cohort ，So I just wonder should I handle this problem before training?

**Comment by BennyStrobes** [comment]
Hi Harry. Very sorry for the delay here. You are correct in that the level of unbalance in the training data does impact model performance. We attempt to quantify the extent of this in supplementary figure S28 of our paper (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7646251/). That said, the best way to combat this is to filter the training data to have a higher fraction of outliers. You can see the filters we used in our paper in supplementary methods section entitled "Applying Watershed to jointly model ASE, splicing, and expression". Let me know if you have any further questions!

**Comment by HarryShen668** [comment]
Dear Ben,
Thanks for your answer which is really helpful for us!As we are interested in your work and desire to use this tool, we evaluated the Watershed performance in our own data,but it seems that we got a bad AUROC about 0.5-0.55. As I didn't find in the paper you provided(could somewhere I didn't notice?),I wonder **what AUROC you got in the data you used**.Such information might be helpful for us to make sure that we are not using the wrong data and to eliminate technical errors .
Thank you very much!

**Comment by HarryShen668** [comment]
Seems that unbalanced evaluate set(with N2pair,not training set),could cause a a bad AUROC .We are a little bit confused about this result .

**Comment by BennyStrobes** [comment]
In general, ROC has been shown to be a poor evaluation metric when the evaluation set is unbalanced (a lot more 0s than 1s), which is naturally the case when using rare variant outlier calls as the evaluation metric. Thus we stuck to AU precision recall curve in our analysis. Have you tried using precision-recall?

I'm happy to chat/have a zoom call to discuss this in more detail. Feel free to email me at bstrober3@gmail.com
